### DISCUSSION TITLE
Help me find tool

### DISCUSSION THREAD

**Discussion by Mushroom-V01** [community/community#134153] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm looking for a software that can help me send automatic messages to Facebook groups

**Comment by AntA001** [comment]
Hootsuite and Postcron are the 2 that I know to be the best. I'm sure there are others too, but I think the 2 I mentioned are some of the most reputable

**Comment by LiteBrite82** [comment]
Hi @Mushroom-V01 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
